src/iconv.jl

```diff
@@ -2,7 +2,7 @@
 
 module iconv
 import Base: close, eof, flush, read, readall, write
-import Base.Libc: errno, strerror, E2BIG, EINVAL, EILSEQ
+import Base.Libc: errno, strerror, E2BIG, EINVAL, EILSEQ, EOVERFLOW
```

Comment:
I'll remove that - I'd added an EILSEQ2 = 92 for mac, and then saw your change, but was afraid that EOVERFLOW might be getting returned also.